git-icdiff

```diff
@@ -11,7 +11,7 @@ if [ -z "$GITPAGER" ]; then
   GITPAGER="${PAGER:-less}"
 fi
 
-if [ "$GITPAGER" = "more" ] || [ "$GITPAGER" = "less" ]; then
+if [ "$GITPAGER%% *" = "more" ] || [ "${GITPAGER%% *}" = "less" ]; then
```

Comment:
I think you're missing some curly braces on the first invocation?

Reply:
```suggestion
if [ "${GITPAGER%% *}" = "more" ] || [ "${GITPAGER%% *}" = "less" ]; then
```
🤦 Sorry, thanks for catching that.